README.md

```diff
@@ -1,10 +1,39 @@
-# Lingo
-
+# Lingo - A build tool for Lingua Franca program
 **Contact:** <tassilo-tanneberger@tu-dresden.de>
 
-Lingo is a build tool for lingua-franca project it will manage dependencies 
-configure build scripts and potentially cross compile for microcontrollers.
+Lingo is a one-stop build tool for the Lingua Franca project. 
+Lingo will manages dependencies, configure build scripts and will potentially cross-compile for embedded platforms.
+
+
+## Getting started
+Lingo is a Rust project and is built with cargo. To install it simply run
+`cargo install --path .`
+
+## The command line interfac
+
+```
+lingua-franca package manager and build tool 0.1.2
+tassilo.tanneberger@tu-dresden.de
+Build system of lingua-franca projects
+
+USAGE:
+    lingo [OPTIONS] <SUBCOMMAND>
+
+OPTIONS:
+    -b, --backend <BACKEND>    force lingo to use the specified backend [default: cli]
+    -h, --help                 Print help information
+    -V, --version              Print version information
+
+SUBCOMMANDS:
+    build     compiling one ore multiple binaries in a lingua-franca package
+    clean     removes build artifacts
+    help      Print this message or the help of the given subcommand(s)
+    init      initializing a lingua-franca project
```

Comment:
We will also need to capitalize these sentences, the casing is inconsistent. I did some of that in #28